search-service/src/test/kotlin/com/egm/stellio/search/csr/service/ContextSourceCallerTests.kt

```diff
@@ -133,6 +131,49 @@ class ContextSourceCallerTests {
         val response = ContextSourceCaller.getDistributedInformation(HttpHeaders.EMPTY, csr, path, emptyParams)
 
         assertTrue(response.isRight())
-        assertNull(response.leftOrNull())
+        assertNull(response.getOrNull())
+    }
+
+    @Test
+    fun `getDistributedInformation should not ask context source for a GEO_JSON`() = runTest {
+        val csr = gimmeRawCSR()
+        val path = "/ngsi-ld/v1/entities/$apiaryId"
+        stubFor(
+            get(urlMatching(path))
+                .willReturn(ok())
+        )
+        val header = HttpHeaders()
+        header.accept = listOf(GEO_JSON_MEDIA_TYPE)
+        ContextSourceCaller.getDistributedInformation(
+            header,
+            csr,
+            path,
+            emptyParams
+        )
+        verify(
+            getRequestedFor(urlPathEqualTo(path))
+                .withHeader(ACCEPT, notContaining("GEO_JSON_MEDIA_TYPE"))
```

Comment:
the value of the header is `application/geo+json` (and not `GEO_JSON_MEDIA_TYPE`)